### DISCUSSION TITLE
Provide additional options to clear logs

### DISCUSSION THREAD

**Discussion by sgodanas** [nxt-dev/nxt_editor#193] [Ideas]
Hi Folks.

At the moment, a "Clear log" button exists that clears the rich output logs. 

I propose that there be three buttons that will enable us to clear rich output, clear raw output and both logs. This will really be helpful during iterating development and testing in the editor.

Thank you.
Satish.

<img width="861" alt="nxt_editor_output_clearing_improvements" src="https://user-images.githubusercontent.com/73957119/121552142-10a63380-c9c5-11eb-8e5f-c7c64b398e9a.png">

**Comment by ImLucasBrown** [comment]
We tried out the buttons, but there's no clean way to have all those buttons.
![image](https://user-images.githubusercontent.com/54835354/124506675-caf54480-dd91-11eb-84d1-720288d22dfb.png)

We opted to add a context menu action. You can also map this action to a hotkey in the hotkey editor. It will clear both the raw and rich log windows. This will be added in the next release.
![image](https://user-images.githubusercontent.com/54835354/124509623-ed8a5c00-dd97-11eb-92cc-3794bdff3d25.png)

**Reply by sgodanas** [reply]
Love it. Thank you :)

**Comment by ImLucasBrown** [comment]
Addressed in [Editor v3.10.0](https://github.com/nxt-dev/nxt_editor/releases/tag/editor_v3.10.0)

**Reply by sgodanas** [reply]
Thank you so much folks :) Going to update NXT today!
